sbi/inference/snl/snl.py

```diff
@@ -53,6 +54,10 @@ def __init__(
         perform inference on the posterior p(theta | x0).
         :param neural_likelihood: Conditional density estimator q(x | theta) in the form of an
         nets.Module. Must have 'log_prob' and 'sample' methods.
+        simulation_batch_size (int): how many parameter sets the simulator takes and converts to data x at
+                the same time. If simulation_batch_size==-1, we simulate ALL parameter sets at the same time.
```

Comment:
NO NEED TO SHOUT. Typically either *bold* or /italics/ are use for emphasis.

Reply:
Bold without markdown escaping is `*bold*` and italics either `/italics/` or `_italics_`.